modules/sdk/gradle-plugins-workspace/src/gradleTest/startDockerContainer/settings.gradle

```diff
@@ -0,0 +1,54 @@
+import com.bmuschko.gradle.docker.tasks.container.DockerLogsContainer
+
+import java.net.InetAddress;
+
+buildscript {
+	dependencies {
+		classpath fileTree(dir: pluginClasspathDir, excludes: ["biz.aQute.bnd-3.5.0.jar", "com.liferay.gradle.util-1.0.33.jar"], include: "*.jar")
+	}
+}
+
+apply plugin: "com.liferay.workspace"
+
+gradle.liferayWorkspace {
+	dockerImageId = InetAddress.getLocalHost().getCanonicalHostName().toLowerCase()
+	dockerContainerId = InetAddress.getLocalHost().getCanonicalHostName().toLowerCase()
+}
+
+gradle.projectsEvaluated {
+	gradle.rootProject.tasks.register("assertScript", DockerLogsContainer) {
```

Comment:
we access gradle.rootProject over and over, so lets assign it to a var

Reply:
Gradle won't allow us to assign `gradle.rootProject` to a script variable due to an error,
Caused by: java.lang.IllegalStateException: The root project is not yet available for build.

Reply:
So inside the `projectsEvaluated` block, declare the root project, so it doesn't have to be repeated